pegasus/sites.v3/code.org/public/teacher-dashboard/index.haml

```diff
@@ -78,9 +78,14 @@ title: <%= I18n.t('dashboard_landing_title').inspect %>
         {templateUrl: '/teacher-dashboard/section', controller: 'SectionDetailController'});
     $routeProvider.when('/sections/:sectionid/student/:studentid',
         {templateUrl: '/teacher-dashboard/student', controller: 'StudentDetailController'});
+    $routeProvider.when('/sections/:sectionid/student/:studentid/script/',
+        {templateUrl: '/teacher-dashboard/student', controller: 'StudentDetailController'});
+    $routeProvider.when('/sections/:sectionid/student/:studentid/script/:scriptid',
+        {templateUrl: '/teacher-dashboard/student', controller: 'StudentDetailController'});
+    // We have legacy /course routes from a time when course was used synonomously with script
```

Comment:
This is one place I made a change that is not just a refactor. I added routes for `/script` and `/script/:scriptid` to be used instead of the `course` equivalents. However, the course routes will still work for the time being (in case teachers have them bookmarked).